src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -14,6 +14,18 @@ def _set_container(self, container):
             self._constraints._container = container
         self.rehint()
 
+    def _set_hidden(self, status, children):
+        for view in self._container._impl.subviews:
+            [view.setHidden_(status) for c in children if c._impl == view]
```

Comment:
This will remove the child remove the child from the view; but it won't modify any of the CSS geometry.